sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -483,6 +483,22 @@ def test_upload_blob_from_pipe(self, **kwargs):
         # Assert
         assert data == blob_data
 
+    @BlobPreparer()
+    @recorded_by_proxy
+    def test_upload_blob_with_tier_specified_cold(self, **kwargs):
```

Comment:
Sorry, I didn't pick up on this earlier, but would you mind moving all the tests in `common_blob` to `block_blob`? Standard access tier is only supported on Block Blobs, so it doesn't really make sense in common (I just learned this a few minutes ago :))